debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9BCUtil.java

```diff
@@ -557,10 +557,10 @@ public static void j9bcutil_dumpRomClass(PrintStream out, J9ROMClassPointer romC
 		/* dump the enclosing method */
 		dumpEnclosingMethod(out, romClass, flags);
 
-		out.append(String.format("Sun Access Flags (0x%s): ", Long.toHexString(romClass.modifiers().longValue())));
+		out.append(String.format("Oracle Access Flags (0x%s): ", Long.toHexString(romClass.modifiers().longValue())));
```

Comment:
Have you verified that no tests are affected by this change?

Reply:
No. I will try it out.

Reply:
The OpenJ9 DDR tests are passing for me locally.

Reply:
As I think about this again, I think (given we're changing things) that it should not refer to 'Sun' or 'Oracle': How about 'Basic Access Flags'?

Reply:
Agreed. I originally added Oracle to match the statement in J9BCUtil.java but I think both should be updated.